lib/dynamoid/undumping.rb

```diff
@@ -156,10 +156,15 @@ def process(value)
 
     class BooleanUndumper < Base
       def process(value)
-        if ['t', true].include? value
-          true
-        elsif ['f', false].include? value
-          false
+        store_as_boolean = if @options[:store_as_native_boolean].nil?
+                             Dynamoid.config.store_boolean_as_native
+                           else
+                             @options[:store_as_native_boolean]
+                           end
+        if store_as_boolean
+          !!value
+        elsif ['t', 'f'].include?(value)
```

Comment:
Would like to make this array a constant within the class.  No need to instantiate again on every call.